job-manager/job-executor/src/test/java/uk/gov/moj/cpp/task/execution/DefaultJobStoreSchedulerPrioritySelectorTest.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.moj.cpp.task.execution;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Mockito.when;
+import static uk.gov.moj.cpp.jobstore.persistence.Priority.HIGH;
+import static uk.gov.moj.cpp.jobstore.persistence.Priority.LOW;
+import static uk.gov.moj.cpp.jobstore.persistence.Priority.MEDIUM;
+
+import uk.gov.moj.cpp.jobstore.persistence.JobStoreConfiguration;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+public class DefaultJobStoreSchedulerPrioritySelectorTest {
+
+    @Mock
+    private RandomPercentageProvider randomPercentageProvider;
+
+    @Mock
+    private JobStoreConfiguration jobStoreConfiguration;
+
+    @InjectMocks
+    private DefaultJobStoreSchedulerPrioritySelector defaultJobStoreSchedulerPrioritySelector;
+
+    @Test
+    public void shouldSelectHighPriorityIfRandomPercentageAboveOrEqualToHighPriorityThreshold() throws Exception {
+
+        when(jobStoreConfiguration.getJobPriorityPercentageHigh()).thenReturn(70);
```

Comment:
I see this is supposed to be percentage .... what happens if I passe value 777 for instance?